cmd/limitometer/main.go

```diff
@@ -9,27 +9,28 @@ import (
 	"syscall"
 	"time"
 
+	"github.com/cerence/azure-request-limitometer/internal/config"
 	"github.com/cerence/azure-request-limitometer/pkg/common"
 	"github.com/cerence/azure-request-limitometer/pkg/outputs"
 
 	"github.com/golang/glog"
 	flag "github.com/spf13/pflag"
 )
 
+var azureClient = common.Client
+
 const (
 	cliName        = "limitometer"
 	cliDescription = "Collects the number of remaining requests in Azure Resource Manager"
 	cliVersion     = "2.0.0"
 )
 
-var config = common.Conf
-var azureClient = common.Client
-
 var (
 	nodename     = flag.String("node", "", "Valid node in the resource group to create compute queries. Environment Variable: NODE_NAME")
 	target       = flag.String("output", "pushgateway", "Target output for the limitometer, supported values are: [influxdb|pushgateway]")
 	mode         = flag.String("mode", "oneshot", "Operational mode for limitometer, supported values are: [oneshot|service]")
 	pollInterval = flag.Int("poll-interval", 60, "Only for 'service' mode: Poll interval for refreshing metrics in seconds")
+	metadata     = flag.String("metadata-config", "loadconfig", "To decide from where to load config, supported values are: [loadconfig|environment]")
```

Comment:
Sorry, I was trying to say that `--config=metadata` and `--config=environment` should be valid. Sorry for the confusion :)

Reply:
Updated the variable as well as name.